### DISCUSSION TITLE
Welcome to threeML Discussions!

### DISCUSSION THREAD

**Discussion by grburgess** [threeML/threeML#433] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by BenCully** [comment]
<img width="736" alt="Screen Shot 2021-06-27 at 10 22 27 PM" src="https://user-images.githubusercontent.com/10446787/123575095-a8ae6600-d796-11eb-96ad-213f14767d70.png">
Maybe a typo "Foundation" ?

**Reply by omodei** [reply]
Fixed

**Comment by Soumya-czti** [comment]
Hii @grburgess , i am currently working on Gamma-Ray Bursts. I have started working with threeML for spectral analysis. I have couple of questions:
1. I have created local model in Xspec, is it possible to invoke that model in threeML? 
2. Can any of the local model of Xspec be used in threeML like the builtin-models of Xspec?

There is one issue I encountered, after successful installation I am unable to used the Xspec model in threeML, though I tried multiple times. I also followed one of your discussion with one of the user, but nothing seems to working in that context.

Can you help me regarding these?

**Reply by grburgess** [reply]
I do not think we support that. Do you mean you have a custom model? You can just create the model in astromodels directly.

**Reply by Soumya-czti** [reply]
Yes, I mean by custom models. I have created  model in Fortran and through the documentations it was not clear how to create it directly in astromodels. If possible can you share any example of model been written in Fortran and used directly in astromodels?

**Reply by grburgess** [reply]
https://threeml.readthedocs.io/en/stable/notebooks/spectral_models.html#Custom-models-in-other-langauges
